index.js

```diff
@@ -60,6 +60,23 @@ function requestAsEventEmitter(opts) {
 			});
 		});
 
+		// this method is not documented, prints a warning if used while
```

Comment:
Looks documented to me: https://nodejs.org/api/http.html#http_response_removeheader_name

Can you find out in which Node.js version it was introduced?